### PR TITLE
Add tests for viva-las-vegas 1.0

### LINKED ISSUE #4: Add tests for viva-las-vegas

### LINKED ISSUE #15: Adapt viva-las-vegas tests for refactored codebase

### PR DESCRIPTION
**Pull request by BehindLoader**
Closes #4 
Closes #15

### CODE REVIEW

File: viva_las_vegas.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    log.info('success!')
+
+def take_a_vacation(browser, log):
+    """
+
+    """
+    log.info('Send leave request')
+    browser.fill('msg', 'хочу в отпуск')
+    if not send_msg_and_try_check_result(browser, LEAVE_REQUEST_EXPECT):
+        print('ERROR')
```
Comment (Sergei-vb) [inline viva_las_vegas.py]: Use `log.error('------------ERROR!!!!!------------')` for such cases.

File: viva_las_vegas.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    yesterday = (datetime.now() - timedelta(days = 1)).strftime('%d.%m')
+    username = os.getenv('SIGN_IN_USERNAME')
+    browser.fill('msg', 'viva reset @%s %s' % (username, yesterday))
+    if not send_msg_and_try_check_result(browser, DEBUG_COMMAND_EXPECT if admin else PERMISSION_DENIED_EXPECT):
+        return False
+    log.info('success!')
+
+def take_a_vacation(browser, log):
+    """
+
```
Comment (Sergei-vb) [inline viva_las_vegas.py]: Add comment for general view.

File: viva_las_vegas_script_tests.py [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+            'Заявка на отпуск пользователя @{0} была отклонена пользователем @{0}.'.format(
+                self.username))
+
+    def _check_cancel_notification_in_channel(self):
+        assert self.get_message_by_number(-2).text == self._dividing_message
+        assert self.check_latest_response_with_retries(
+            'Пользователь @{0} отменил заявку на отпуск пользователя @{0}.'.format(
+                self.username))
+
+    def test_viva_las_vegas_with_approve(self):
```
Comment (eugulixes) [inline viva_las_vegas_script_tests.py]: `test_sending_request_and_approving_it`

File: viva_las_vegas_script_tests.py [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+        self.choose_general_channel()
+        self._send_vacation_request()
+        self._input_start_date()
+        self._input_end_date()
+        self._confirm_dates()
+        self._approve_vacation()
+        self._cancel_vacation()
+
+    def test_viva_las_vegas_with_reject(self):
```
Comment (eugulixes) [inline viva_las_vegas_script_tests.py]: `test_sending_request_and_rejecting_it`

**Pull request merged**